symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/properties/CorsProperties.java

```diff
@@ -0,0 +1,21 @@
+package com.symphony.bdk.app.spring.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Configuration Properties for enabling the CORS support to accept requests coming from the extension app.
+ */
+@Getter
+@Setter
+public class CorsProperties {
+
+  /**
+   * CORS handling for the specified path patterns.
+   */
+  private List<String> allowedOrigins = Collections.singletonList("/**");
```

Comment:
Since we are here, let's support [other properties](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/servlet/config/annotation/CorsRegistration.html) 😄 
- `allowCredentials` (boolean)
- `allowedHeaders` (list string)
- `exposedHeaders` (list string)
- `allowedMethods` (list string)
- `exposedHeaders` (list string)

Reply:
Even if not used at the moment for the example, developers might need to configure it in the future